### DISCUSSION TITLE
Hi

### DISCUSSION THREAD

**Discussion by niltonheck** [niltonheck/test-githubactions#39] [General]
How to get started?

**Comment by glime-ai** [comment]
Sure, I can help you get started with both Kinde and AsteroidKit.

For Kinde, you can follow these steps:

1. Sign up for a business and get a Kinde domain [here](https://app.kinde.com/register).
2. Get a starter kit from [GitHub](https://github.com/kinde-starter-kits).
3. Add an application in Kinde. You can do this by going to Settings > Applications.
4. Get your app keys and add them to the .env file that came with the starter kit.
5. Start your local server and view your project in a browser. For example, if you're using NextJS, the command is `npm run dev`.
6. Once your project is running in the browser, select Sign up to register your first user.
7. When you're done, view the user in the main Users page in Kinde.

For AsteroidKit, you can follow these steps:

1. Create an account [here](http://app.asteroidkit.com/).
2. Install the SDK library using `npm install asteroidkit wagmi`.
3. Configure the SDK with your project ID.
4. Add the connect button to your app using the `ConnectButton` component.
5. Verify your setup by logging in to your application and checking that you see the logged in user in the dashboard.

You can find more detailed instructions in the [Kinde documentation](https://kinde.com/docs/connect-to-kinde/start-with-starter-kit) and the [AsteroidKit documentation](https://docs.asteroidkit.com/getting-started).

Let me know if you need any more help!
